src/packages/@ncigdc/components/SetActions.js

```diff
@@ -88,6 +88,7 @@ export default enhance(
                     type={type}
                     displayType={displayType}
                     CreateSetButton={CreateSetButton}
+                    selectedIds={selectedIds}
```

Comment:
looks like `SaveSetModal` takes `setName` as a prop so you could pass `setName={(selectedIds || []).length ? `Custom ${displayType} Set` : '' }` here instead and avoid having to additionally pass selectedIds to `SaveSetModal`? Doesn't super matter but it's less code, and keeps SaveSetModal having to know about selected ids, which are already in the filters.